README.md

```diff
@@ -2723,6 +2723,13 @@ IEEE Robotics and Automation Letters (RA-L), 2021. [Dataset](http://rpg.ifi.uzh.
 - [EV-IMO Event based Independently Moving Objects dataset](https://better-flow.github.io/evimo/) associated to the paper [EV-IMO: Motion Segmentation Dataset and Learning Pipeline for Event Cameras](#Mitrokhin19iros) ([motion vector flow](https://github.com/better-flow/evimo/blob/master/tools/evimo_flow.py) added Jan 2022)
 - [Chaney et al., CVPRW 2023](#Chaney23cvprw), *M3ED: Multi-Robot, Multi-Sensor, Multi-Environment Event Dataset*
 
+## Eye Tracking
+- [Chen et al., CVPRW 2024](#Chen23biocas), *3ET: Efficient Event-based Eye Tracking using a Change-Based ConvLSTM Network*
+- [Bonazzi et al., CVPRW 2024](#Bonazzi24cvprw), *Retina: Low-Power Eye Tracking with Event Camera and Spiking Hardware*
```

Comment:
the 2nd one does not appear to recognize the link
![image](https://github.com/uzh-rpg/event-based_vision_resources/assets/8024432/f31c5f5c-4162-4759-97ff-31e5461e0589)

(probably because the changes in the other pull request were not incorporated in this one. Let's see what happens after the merge.

Reply:
It works fine. Confirmed. Thanks.